scripts/add_directory.js

```diff
@@ -162,19 +162,20 @@ function submitBatch(n) {
     buttons.toggleClass("disabled");
     $(".glyphicon", buttons).toggleClass("hidden");
     // eventually this will make an actual post request
-    setTimeout(function() {
-        res = [{result: "PASS", err_msg: null, img_errs: []},
-            {result: "ERROR", err_msg: "Unhashable images detected", img_errs: ["02_98.png", "03_98.png"]},
-            {result: "FAIL", err_msg: "Database unreachable", img_errs: []}
-            ][Math.floor(Math.random() * 100) % 3];
+    $.get("/test-add", batchReq, function(res) {

```

Comment:
We will need to change this from `test-add` to something reasonable later down the line :+1: